orchagent/Makefile.am

```diff
@@ -53,11 +55,12 @@ orchagent_SOURCES = \
             flexcounterorch.cpp \
             watermarkorch.cpp \
             policerorch.cpp \
-            chassisorch.cpp
+            chassisorch.cpp \
+            debugcounterorch.cpp
 
 orchagent_CFLAGS = $(DBGFLAGS) $(AM_CFLAGS) $(CFLAGS_COMMON) $(CFLAGS_SAI)
 orchagent_CPPFLAGS = $(DBGFLAGS) $(AM_CFLAGS) $(CFLAGS_COMMON) $(CFLAGS_SAI)
-orchagent_LDADD = -lnl-3 -lnl-route-3 -lpthread -lsairedis -lswsscommon -lsaimeta -lsaimetadata
+orchagent_LDADD = -lnl-3 -lnl-route-3 -lpthread -lsairedis -lswsscommon -lsaimeta -lsaimetadata $(utildir)/flex_counter/libflexcounter.la $(utildir)/debug_counter/libdebugcounter.la
```

Comment:
>libdebugcounter [](start = 163, length = 15)

If it is not shared, we could directly use source code instead of creating a library.